docs/man/Dockerfile.5.md

```diff
@@ -143,6 +143,21 @@ A Dockerfile is similar to a Makefile.
   **CMD** executes nothing at build time, but specifies the intended command for
   the image.
 
+**LABEL**
+  -- `LABEL <key>[=<value>] [<key>[=<value>] ...]`
+  The **LABEL** instruction adds metadata to an image. A **LABEL** is a
+  key-value pair. To include spaces within a **LABEL** value, use quotes and
+  blackslashes as you would in command-line parsing.
```

Comment:
@TomasTomecek that's "dark matter" I think :smile: Do you want to make a pull-request to fix that?